benchmarks/benchmark_bundle_method.jl

```diff
@@ -0,0 +1,31 @@
+using Manopt, Manifolds, Random, QuadraticModels, RipQP, BenchmarkTools, Test
```

Comment:
Do we want to keep this file here? I am not yet sure how to provide benchmarks.

Reply:
This folder should be removed, too.

Reply:
Did that already :)

Reply:
These files are so old I completely forgot they existed.